quizzes/admin.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib import admin
```

Comment:
Insert a docstring or in-line comments to explain the function of the contents of this file. We discussed it verbally and it all makes sense, just be sure to document.

Reply:
Will do!